src/autopas/tuning/AutoTuner.h

```diff
@@ -382,5 +382,21 @@ class AutoTuner {
    * CSV logger for all samples collected during a tuning phase.
    */
   TuningDataLogger _tuningDataLogger;
+
+  /**
+   * Is set to true during a tuning phase.
+   */
+  bool _isTuning{false};
+
+  /**
+   * Is only set to true in the iteration where a tuning phase ends. This is done in tuneConfiguration() and set back to
+   * false in bumpIterationCounters()
```

Comment:
This means that from the moment the optimal configuration until the next bump of counters this is true?
Then I suggest this formulation:
```suggestion
   * Is only set to true for the last iteration of a tuning phase ends.
   * Specifically, from when tuneConfiguration() selects the optimum until it is reset to false in bumpIterationCounters().
```